src/components/name-text.jsx

```diff
@@ -123,7 +123,7 @@ function NameText({
       )}
       {showAcct && (
         <>
-          <br />
+          {' '}
```

Comment:
This change seems like could cause some layout issues.

Reply:
Seems to me that it's necessary to prevent them, actually!

With `<br>`:

![](https://github.com/user-attachments/assets/bcbf903f-00b2-4a66-af2c-76859ed98d4c)

With `{' '}`:

![image](https://github.com/user-attachments/assets/be1856f9-563c-4d27-a4f7-13e1e54f1c07)

Reply:
The `<br>` is used for the large-size post. Maybe need to rename or add another prop to allow both cases.